src/operation.rs

```diff
@@ -244,12 +334,20 @@ impl Init {
     /// We will, however, refuse to install any TALs until `accept_arin_rpa`
```

Comment:
This comment seems outdated, or at least misplaced with the new opt-in construction.

Reply:
Removed.